docs/contributing/releasing.md

```diff
@@ -0,0 +1,56 @@
+# Releasing
+
+Argo Image Updater is released in a 2 step automated fashion using GitHub actions. The release process takes about 20 minutes, sometimes a little less, depending on the performance of GitHub Actions runners.
+
+Releases can only be done  by people that have write/commit access on the Argo Image Updater GitHub repository.
+
+## Introduction
+
+First install GoLang and Git locally on your workstation. Then create a release branch:
+
+```
+git clone git@github.com:argoproj-labs/argocd-image-updater.git
+git branch -b release-0.13
+git push origin release-0.13
+```
+
+The release name is just an example. You should use the next number from the [previous release](https://github.com/argoproj-labs/argocd-image-updater/releases). Make sure that the branch is named as `release-X.XX` though.
+
+Also note that `origin` is just an example. It should be the name of your remote that holds the upstream branch of Argo Image updater.
+
+Finally run
+
+```
+./hack/create-release-pr.sh origin
+```
```

Comment:
It should be noted that this script needs the GitHub CLI. It also uses something called `semver-cli`, which I've never heard of.

Reply:
I added the requirements in the documentation page itself.